src/main/java/org/gridsuite/securityanalysis/server/dto/LimitViolationDTO.java

```diff
@@ -43,7 +43,7 @@ public static LimitViolationDTO toDto(AbstractLimitViolationEntity limitViolatio
     public List<String> toCsvRow(Map<String, String> translations) {
         List<String> csvRow = new ArrayList<>();
         csvRow.add(this.getLimitType() != null ? CsvExportUtils.translate(this.getLimitType().name(), translations) : "");
-        csvRow.add(CsvExportUtils.replaceNullWithEmptyString(this.getLimitName()));
+        csvRow.add(CsvExportUtils.replaceNullWithEmptyString(CsvExportUtils.translate(this.getLimitName(), translations)));
```

Comment:
I could be nice to add a test covering this

Reply:
Done